tm2/pkg/bft/state/eventstore/types/config.go

```diff
@@ -25,5 +25,6 @@ func (c *Config) GetParam(name string) any {
 func DefaultEventStoreConfig() *Config {
 	return &Config{
 		EventStoreType: null.EventStoreType,
+		Params:         make(EventStoreParams),
```

Comment:
It looks unusual to see `make` with only one argument. I don't think it matters too much; just caught my eye is all.